osquery/tables/system/posix/load_average.cpp

```diff
@@ -10,17 +10,24 @@
 
 #include <stdlib.h>
 
+#include <array>
+
 #include "osquery/core/conversions.h"
 #include <osquery/tables.h>
 
 namespace osquery {
 namespace tables {
 
+// TODO: Use std::stringview when we drop requirement for everything other
```

Comment:
I'm not sure we need this TODO, can you accomplish this with `boost::string_ref`?

Reply:
I'm not familiar with it. Not sure it supports constexpr.

Reply:
Boost has a [stringview implementation](http://www.boost.org/doc/libs/1_64_0/boost/utility/string_view.hpp) which is the same as the one in C++17.
Let's do that :)

Reply:
I haven't checked if it compiles yet. I'll do so when I get home.
In any case, I think we should use std::stringview if it is available instead of the boost version but this is good enough for now.